test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -175,6 +175,80 @@ public static void Main()
                 .And
                 .HaveStdOutContaining("Hello from a netcoreapp2.0.!");
         }
+        
+        //Note: Pre Netcoreapp2.0 standalone activation uses renamed dotnet.exe
+        //      While Post 2.0 we are shifting to using apphost.exe, so both publish needs to be validated
+        [Theory]
+        [InlineData("win-arm")]
+        [InlineData("win8-arm")]
+        [InlineData("win81-arm")]
+        [InlineData("win10-arm")]
```

Comment:
I think that's a good idea, the more we cover the better - the only thing is I don't have an arm64 device to test this on, do we know if the list of files present on arm 32-bit devices should be the same as the 64-bit devices?